src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -1496,6 +1448,26 @@ export class StudyViewPageStore {
         this._geneFilterSet.delete(chartUniqueKey);
     }
 
+    @autobind
+    @action
+    removeGenomicProfileFilter(toBeRemoved: string) {
+        let genomicProfilesFilter = toJS(this.genomicProfilesFilter) || [];
```

Comment:
toJS will return falsy if passed something false?

Reply:
toJS would return `undefined` if `this.genomicProfilesFilter` is `undefined`